reviewrot/basereview.py

```diff
@@ -82,6 +87,64 @@ def check_request_state(self, created_at,
                 raise ValueError("Invalid duration type: %s" % duration)
         return True
 
+    def _decode_response(self, response):
```

Comment:
Can we set the Accepts request header to "application/json" to avoid this? Under what scenarios does gerrit reply with different formats?